pkg/testing/v1/revision.go

```diff
@@ -182,10 +182,3 @@ func WithRevisionLabel(key, value string) RevisionOption {
 		config.Labels[key] = value
 	}
 }
-
-// WithContainerStatuses sets the .Status.ContainerStatuses to the Revision.
-func WithContainerStatuses(containerStatus []v1.ContainerStatuses) RevisionOption {
```

Comment:
Any reason why you couldn't use this function and made a private copy?

Reply:
WithContainerStatuses was used only in `table_test.go` file so made it as a private

Reply:
Reverted back changes I mean again moved back to `revision.go` file because its been used in `deploy_test.go` and `table_test.go`